scripts/globals/status.lua

```diff
@@ -80,6 +80,8 @@ SUBEFFECT_HP_DRAIN          = 21;  -- 1-10101      43
 SUBEFFECT_MP_DRAIN          = 22;  -- This is correct animation
 SUBEFFECT_TP_DRAIN          = 22;  -- Verified this should look exactly like Aspir Samba.
 SUBEFFECT_HASTE             = 23;
+SUBEFFECT_CHOKE             = 24;
```

Comment:
If you are certain the ID is correct, add the subeffect in source as well please https://github.com/DarkstarProject/darkstar/blob/c85204b161ba608cca351692d89bf26d51b811e7/src/map/entities/battleentity.h#L267

Note these are not arbitrary so you better have tested it locally.